src/callback.html

```diff
@@ -15,6 +15,12 @@
   justify-content: center;
   align-items: center;
 }
+@media (prefers-color-scheme: dark) {
+  body, html {
+    background-color: #131313;
+    color: #ffffff;
```

Comment:
I've taken some colours from the neon console, please let me know if I need to adjust them to match the Neon style.

Reply:
On the console we use
```
background-color: #191919;
color: #bfbfbf;
```

Reply:
A-ha, thanks!

While we're here, do we need to update the light theme (currently we use just black and white)?

Reply:
For the light theme it's
```
background-color: #ffffff;
color: #2d374c;
```